src/components/radio/RadioButtonField.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import * as React from 'react';
+import type { FieldProps } from 'formik';
+
+import RadioButton from './RadioButton';
+import type { RadioButtonProps } from './RadioButton';
+
+type Props = RadioButtonProps & FieldProps;
+
+const RadioButtonField = ({ field, form, value, ...rest }: Props) => {
+    const { value: fieldValue, ...fieldRest } = field;
+    return <RadioButton {...fieldRest} {...rest} value={value} isSelected={value === fieldValue} />;
```

Comment:
It seems like this could be accomplished by using `RadioButtonGroup` with a unique name?